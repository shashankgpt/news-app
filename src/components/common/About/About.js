import React from 'react'

function About() {
    return (
        <aside className="col-md-4 blog-sidebar">
        <div className="p-3 mb-3 bg-light rounded">
          <h4 className="font-italic">About</h4>
          <p className="mb-0">Etiam porta <em>sem malesuada magna</em> mollis euismod. Cras mattis consectetur purus sit amet fermentum. Aenean lacinia bibendum nulla sed consectetur.</p>
        </div>

        {/* <div className="p-3">
          <h4 className="font-italic">Archives</h4>
          <ol className="list-unstyled mb-0">
            <li><a href="w">March 2014</a></li>
            <li><a href="w">February 2014</a></li>
            <li><a href="w">January 2014</a></li>
            <li><a href="w">December 2013</a></li>
            <li><a href="w">November 2013</a></li>
            <li><a href="w">October 2013</a></li>
            <li><a href="w">September 2013</a></li>
            <li><a href="w">August 2013</a></li>
            <li><a href="w">July 2013</a></li>
            <li><a href="w">June 2013</a></li>
            <li><a href="w">May 2013</a></li>
            <li><a href="w">April 2013</a></li>
          </ol>
        </div>

        <div className="p-3">
          <h4 className="font-italic">Elsewhere</h4>
          <ol className="list-unstyled">
            <li><a href="w">GitHub</a></li>
            <li><a href="w">Twitter</a></li>
            <li><a href="w">Facebook</a></li>
          </ol>
        </div> */}
      </aside>

    )
}

export default About
