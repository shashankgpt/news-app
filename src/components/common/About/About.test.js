import React from 'react';
import {mount} from 'enzyme';
import About from './About';

const setup = (props={}) => {
    return mount(<About {...props} />)
}

describe('Testing About', () => {
    let wrapper;
    beforeEach(()=> {
        wrapper = setup()
    });

    it('Check snapshot for about', () => {
        expect(wrapper).toMatchSnapshot();
    })
})