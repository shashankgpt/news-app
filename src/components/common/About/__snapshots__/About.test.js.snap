// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing About Check snapshot for about 1`] = `ReactWrapper {}`;
